tests/unit/commands/test_borgmatic.py

```diff
@@ -422,6 +422,26 @@ def test_run_actions_runs_rcreate():
         )
     )
 
+def test_run_actions_adds_log_file_to_hook_context():
```

Comment:
I'm not seeing any asserts/expectations that log file is actually making its way into `hook_context` though..?